docs/sphinx/user/inputs_Boundary_conditions.rst

```diff
@@ -87,3 +87,32 @@ The most applicable use case for this boundary condition is with the
 :ref:`amrwind-abl-bndry-io` for flows that change directions
 across the vertical coordinate or with time.
 The work to integrate this condition with the ABL class is under progress.
+
+Dynamic wall model (Wave model)
```

Comment:
I like documentation. Can you also add a brief item linking to this section in `features.rst` ?

Reply:
Sure, I will do that and I will also add some documentation in theory.rst to show some of the equations of the model